app/containers/Admin/ProductNaicsCode/AllowableProductsTableRow.tsx

```diff
@@ -0,0 +1,88 @@
+import {faCheck} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import DeleteConfirmationModal from 'components/Admin/DeleteConfirmationModal';
+import deleteProductNaicsCodeMutation from 'mutations/product_naics_code/deleteProductNaicsCodeMutation';
+import React, {useState} from 'react';
+import {Button} from 'react-bootstrap';
+import {createFragmentContainer, graphql, RelayProp} from 'react-relay';
+import {AllowableProductsTableRow_productNaicsCode} from '__generated__/AllowableProductsTableRow_productNaicsCode.graphql';
+
+interface Props {
+  relay: RelayProp;
+  productNaicsCode: AllowableProductsTableRow_productNaicsCode;
+  naicsCodeId: string;
+}
+
+export const AllowableProductsTableRowComponent: React.FunctionComponent<Props> = ({
+  relay,
+  productNaicsCode,
+  naicsCodeId
+}) => {
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+
+  const deleteProductNaicsCode = async () => {
+    const {environment} = relay;
+
+    await deleteProductNaicsCodeMutation(
+      environment,
+      naicsCodeId,
+      productNaicsCode,
+      'AllowableProducts_productNaicsCodesByNaicsCodeId'
+    );
+  };
+
+  const deleteModalContent = {
+    deleteName: 'allowed Product',
```

Comment:
```suggestion
    deleteName: 'allowed product',
```


<img width="560" alt="Screen Shot 2021-04-06 at 10 22 52 AM" src="https://user-images.githubusercontent.com/5522075/113752900-580cfc80-96c2-11eb-820e-2acdf1b1b4c5.png">